wallet-webapps-common/src/main/webapp/vue-app/walletCommon.js

```diff
@@ -42,6 +42,10 @@ import WalletSettings from './wallet-settings/components/WalletSettings.vue';
 import WalletNotificationAlert from './components/WalletNotificationAlert.vue';
 import MetamaskWarnings from './components/MetamaskWarnings.vue';
 import WalletWelcomeScreen from './components/WalletWelcomeScreen.vue';
+import WalletOverview from '../../../../../wallet-webapps/src/main/webapp/vue-app/wallet-overview/components/WalletOverview.vue';
+import WalletOverviewDrawer from '../../../../../wallet-webapps/src/main/webapp/vue-app/wallet-overview/components/WalletOverviewDrawer.vue';
+import WalletOverviewRewardItem from '../../../../../wallet-webapps/src/main/webapp/vue-app/wallet-overview/components/WalletOverviewRewardItem.vue';
+import WalletOverviewRewardPluginItem from '../../../../../wallet-webapps/src/main/webapp/vue-app/wallet-overview/components/WalletOverviewRewardPluginItem.vue';
```

Comment:
IMO, should be moved in commons components. WDYT?

Reply:
Done!